File: terkey.php
#!/usr/bin/env php
<?php
/*
File: terkey.php
Author: RandsX
Description: Termux Shortcut Key
*/

class Terkey {

  const dir = "/data/data/com.termux/files/home/.termux";

  public function __construct() {
    $this->banner();
    @mkdir(self::dir);
    $this->setup();
    print system("termux-reload-settings");
    print system("clear");
    $this->banner();
    print "Shortcut Success Installed\n";
  }

  protected function banner() {
    print system("clear");
    print "
  ______          __ __
 /_  __/__  _____/ //_/__  __  __
  / / / _ \/ ___/ ,< / _ \/ / / /
 / / /  __/ /  / /| /  __/ /_/ /
/_/  \___/_/  /_/ |_\___/\__, /
         22XploiterCrew /____/\n\n";
  }

  public function setup() {
    print "Settings Up The Shortcut\n";
    $keys = "extra-keys = [['ESC','/','-','clear','UP','exit','HOME'],['TAB','CTRL','ALT','LEFT','DOWN','RIGHT','END']]";
    $open = fopen(self::dir."/termux.properties", "w");
    fwrite($open, $keys);
    fclose($open);
  }
}

$run = new Terkey;

File: terkey.py
'''
File: terkey.py
Author: RandsX
Description: Termux Shortcut Key
'''
from threading import Thread
import os as termux
logo = """
  ______          __ __
 /_  __/__  _____/ //_/__  __  __
  / / / _ \/ ___/ ,< / _ \/ / / /
 / / /  __/ /  / /| /  __/ /_/ /
/_/  \___/_/  /_/ |_\___/\__, /
         22XploiterCrew /____/
"""

def banner():
  termux.system("clear")
  print(logo)

def init():
  # TODO: write code...
  try:
    termux.mkdir('/data/data/com.termux/files/home/.termux')
  except:
    pass
  keys = "extra-keys = [['ESC','/','-','clear','UP','exit','HOME'],['TAB','CTRL','ALT','LEFT','DOWN','RIGHT','END']]"
  open('/data/data/com.termux/files/home/.termux/termux.properties','w').write(keys)
  termux.system('termux-reload-settings')
  
if __name__ == '__main__':
  # TODO: write code..
  banner()
  thread = Thread(target=init)
  thread.start()
  print("Settings Up The Shortcut")
  termux.system("sleep 2")
  banner()
  print("Shortcut Sucess Installed")